advocacy_docs/partner_docs/ThalesCipherTrustManager/02-PartnerInformation.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: 'Partner Information'
+description: 'Details of the Partner'
+
+---
+| &nbsp;     | &nbsp; |
+| ----------- | ----------- |
+| **Partner Name**      |    Thales    |
+| **Web Site**      |   https://cpl.thalesgroup.com/   |
+| **Partner Product**   |   CipherTrust Manager     |
+| **Version**   |   2.11    |
+| **Product Description**      |  Thales CipherTrust Manager simplifies key lifecycle management tasks for your EDB Postgres Advanced Server 15.2 instances.  With Thales CipherTrust Manager you have secure key generation, rotation, destruction, and import and export abilities.   |
```

Comment:
should be saying 15.2 and above int his and other guides as it will work with the newer ones as well.